### PR TITLE
fix nightly builds

### PR DESCRIPTION
**Pull request by mikemhenry**
Namespace changes are coming in openmm, they are backwards compatible if we modify our import slightly.

https://github.com/openmm/openmm/issues/3022

### CODE REVIEW

**Comment by jchodera** [comment]
As @mikemhenry notes, we can easily fix the failures like this:
```
___________ ERROR collecting perses/tests/test_topology_proposal.py ____________
ImportError while importing test module '/home/runner/work/perses/perses/perses/tests/test_topology_proposal.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/share/miniconda/envs/test/lib/python3.8/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
perses/tests/test_topology_proposal.py:18: in <module>
    from perses.rjmc.topology_proposal import SmallMoleculeSetProposalEngine
perses/rjmc/__init__.py:1: in <module>
    from perses.rjmc import geometry, coordinate_numba, coordinate_tools, topology_proposal
perses/rjmc/topology_proposal.py:13: in <module>
    import openmoltools.forcefield_generators as forcefield_generators
/usr/share/miniconda/envs/test/lib/python3.8/site-packages/openmoltools/__init__.py:26: in <module>
    from openmoltools import amber_parser, system_checker, utils, packmol, openeye, amber, cirpy, gromacs, schrodinger
/usr/share/miniconda/envs/test/lib/python3.8/site-packages/openmoltools/amber_parser.py:4: in <module>
    import simtk.openmm.app.element as element
E   ModuleNotFoundError: No module named 'simtk.openmm.app.element'
```
according to https://github.com/openmm/openmm/issues/3022#issuecomment-778410047, by changing instances of
```
from simtk.openmm.app.element import Element
``` 
to
```python
from simtk.openmm.app import Element
```

**Review by jchodera — APPROVED** [review]
Approved! Is this ready to merge?

**Comment by mikemhenry** [comment]
It depends on if I can get `parmed` to accept a PR that I'm going to submit later today. I might need to include a patch so that parmed also works with openmm nightly.

**Comment by mikemhenry** [comment]
Nightly builds are passing :tada: they are still marked as not required but they work now! In order for them to run, right now the CI pulls in two forks I have for some upstream packages, they can be removed once upstream merges in my changes and cuts a new release.

**Comment by mikemhenry** [comment]
Just asking for re-approval because I wanted to make sure you couldn't think of a better way to handle the changes that need to happen upstream. I'm not too worried since this is only for the nightly builds so it isn't like we are going to vendor or support these forks. I'm keeping an eye on parmed, and I didn't want to have to wait for the openmoltools pr to get approved, merged, then end up on conda-forge

File: .github/workflows/CI.yaml
Comment (jchodera) [inline .github/workflows/CI.yaml:69]: Let's create an issue to remind us to fix this at the next release!

**Review by jchodera — APPROVED** [review]
Looks great! Thanks!

**Pull request merged**
